resources/ext/ezbake.conf

```diff
@@ -20,22 +19,42 @@ ezbake: {
                preinst: ["install --group={{user}} --owner={{user}} -d /var/lib/puppet/jruby-gems",
                          "mkdir -p /etc/puppet/manifests"],
```

Comment:
not sure what this is going to shake out to be based on code/config separation. /etc/puppetlabs/agent/manifests is probably a sane choice in the meantime.